internal/storage/db/internal/db.go

```diff
@@ -349,25 +382,29 @@ func (s *dbStorage) Delete(ctx context.Context, ids ...namer.ModuleID) error {
 	return nil
 }
 
-func (s *dbStorage) GetPolicies(ctx context.Context) ([]*policy.Wrapper, error) {
-	res, err := s.db.From(PolicyTbl).Executor().ScannerContext(ctx)
+func (s *dbStorage) ListPolicyIDs(ctx context.Context) ([]string, error) {
+	res, err := s.db.Select(goqu.C(PolicyTblKindCol), goqu.C(PolicyTblNameCol), goqu.C(PolicyTblVerCol)).
```

Comment:
Might be worth creating a composite index or a computed column for this in the database. That would help with efficient batch retrieval from `GetPolicy` calls as well.

Reply:
Annoyingly, computed column would require a database migration step. So let's not do that. Creating a composite index would be fine though. If it doesn't exist, the query would still work, albeit a little slower.